backend/controllers/appointmentsService.js

```diff
@@ -32,6 +37,12 @@ const toAppointmentDetailsSummary = ({
     recurring: properCaseName(recurring),
     ...recurringInformation,
   }
+
+  if (isVideoLinkBooking(appointmentType)) {
+    ;['appointmentType', 'recurring'].forEach(e => delete appointmentInfo[e])
+  }
```

Comment:
It made a bit more sense to customise for VLB, rather than have a check for if the appointment isn't VLB.